scripts/mass-deletion/mass-deletion.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+if [[ -n "$GRIDKEY" || -n "$GRIDDOMAIN" ]]; then
+  echo "make sure to set env vars GRIDKEY and GRIDDOMAIN to run this script"
+  exit 1
+fi
+
+del=todelete.txt
+prog=progress.txt
+com="complete.txt"
+errs=errors.txt
+
+touch $prog
+touch $com
+touch $errs
+
+all="$(wc -l $del | awk '{ print $1 }' )"
+ndone=0
+
+skipping=1
+last="$(cat $prog)"
+
+
+
+while read id; do
+  ndone=$((ndone + 1))
+  if [[ $skipping = 1 && -n "$last" && $id != $last ]]; then
+    continue
+  elif [[ $skipping = 1 && -n "$last" ]]; then
```

Comment:
it took me a minute to work out why skipping was initially set to 1. I take it that it's because we're looking through the ids until we find the first one after the 'prog' value (if there's one set) and only then do we start processing them? this seems v sensible but might benefit from a brief comment to explain the pattern?

Reply:
thanks, added